Configurations/unix-Makefile.tmpl

```diff
@@ -399,7 +399,7 @@ clean: libclean
 	$(RM) -r test/test-runs
 	$(RM) openssl.pc libcrypto.pc libssl.pc
 	-$(RM) `find . -type l \! -name '.*' -print`
-	$(RM) $(TARFILE)
+	$(RM) '$(TARFILE)'
```

Comment:
TARFILE has no spaces?

Reply:
`make TARFILE="../OpenSSL archive/openssl-20190916.tar.gz"`

Reply:
I know, but it's already quoted in a place and generally speaking, passing a variable without quoting to `rm` is playing with fire.

Reply:
Okay.